modules/tensor_mechanics/src/base/TensorMechanicsApp.C

```diff
@@ -342,6 +343,8 @@ TensorMechanicsApp::associateSyntax(Syntax & syntax, ActionFactory & action_fact
   syntax.registerActionSyntax("EmptyAction", "BCs/Pressure");
   syntax.registerActionSyntax("PressureAction", "BCs/Pressure/*");
 
+  syntax.registerActionSyntax("GeneralizedPlaneStrainAction", "GeneralizedPlaneStrain/*");
```

Comment:
@permcody Has the decision been made to put everything in Modules under top-level blocks named after the modules? That would affect a lot more than just this.